Makefile

```diff
@@ -93,8 +92,7 @@ run: server
 	./$(APP_NAME)
 
 test: vendor glide.lock
-	go test $(GOFILES)	# TODO: move this into a sub-test inside test/*.sh
```

Comment:
I did not realize this when the original patch was merged but we need "go test" in topdir of repo. This is required to run the unit tests in apps/glusterfs/*

Reply:
Er, this does not seem to make a lot of sense to me: You can always change dirs from the actual test script. test scripts should go into tests/ , imho. :-)

Reply:
Yup, as long as tests in directories other than tests/* are called, I am happy.